plugins/filters/ndmean/ndmean.h

```diff
@@ -0,0 +1,19 @@
+//
+// Created by sosca on 12/03/2021.
+//
```

Comment:
Please add the typical header for Blosc Developers.

Reply:
BTW, it would be really great if you do a more broader overview so that all the headers in the repo are consistent?

A standard header looks like this:

```
/*********************************************************************
  Blosc - Blocked Shuffling and Compression Library

  Copyright (C) 2021  The Blosc Developers <blosc@blosc.org>
  https://blosc.org
  License: BSD 3-Clause (see LICENSE.txt)

  See LICENSE.txt for details about copyright and rights to use.
**********************************************************************/
```